bishnu-suyel/Druid-Final-Project
# TDFP-12:Update landing page with dummy contents

Related Ticket: TDFP-8

In This PR:
Drupal site:
Adds new content type name home with image and text.
Creates content and add contents in the content type home.
Create a views  called home page and add those  content types in it.
Checks accepted request formats to json inside the formats.
Adds rest export to that views and add path /home.
Grants permission to view the published content.

React site:
Updates the landing page with hamburger menu and logo on the top.
Creates layout and adds footer at the end.
Adds some dummy content  in the landing page.
Adds dummy page about us and contact.

To Test:
Pull the code to your local main branch.
Run the command "lando drush cim".
Run the command "lando db-import database.sql"
Run the command "lando start"  in the root of the folder and "npm run dev" in the frontend folder and see the changes.

Please review the changes and provide feedback.
